doc/changelog.md

```diff
@@ -2,6 +2,62 @@
+- Updated runtime/lib/backend/openqasm/OpenQasmBuilder.hpp to include the two gates so that the backend can look them up from OpenQasm;
+- Updated runtime/lib/backend/common/Utils.hpp to include the PSWAP and ISWAP gates.
+
+A circuit like 
+```python
+import pennylane as qml
+from catalyst import *
+
+dev = qml.device("braket.local.qubit", wires=2, shots=100)
+
+@qjit
+@qml.qnode(dev)
+def f(x: float):
+    qml.Hadamard(0)
+    qml.PSWAP(x, wires=[0, 1])
+    qml.ISWAP(wires=[1, 0])
+    return qml.probs()
+```
+would no longer decompose the PSWAP and ISWAP gates into braket backend's SWAPs and CNOTs and Hadamards. Instead it would just call braket backend's PSWAP and ISWAP:
+
+```python
+{ lambda ; a:f64[]. let
+    b:f64[4] = func[
+      call_jaxpr={ lambda ; c:f64[]. let
+           = qdevice[
+            rtd_kwargs={'shots': 100, 'device_type': 'braket.local.qubit', 'backend': 'braket_sv'}
+            rtd_lib=/home/paul/catalyst/frontend/catalyst/utils/../../../runtime/build/lib/librtd_openqasm.so
+            rtd_name=OpenQasmDevice
+          ] 
+          d:AbstractQreg() = qalloc 2
+          e:AbstractQbit() = qextract d 0
+          f:AbstractQbit() = qinst[op=Hadamard qubits_len=1] e
+          g:AbstractQbit() = qextract d 1
+          h:AbstractQbit() i:AbstractQbit() = qinst[op=PSWAP qubits_len=2] f g c
+          j:AbstractQbit() k:AbstractQbit() = qinst[op=ISWAP qubits_len=2] i h
+          l:AbstractObs(num_qubits=2,primitive=compbasis) = compbasis k j
+          m:f64[4] = probs[shape=(4,)] l
+          n:AbstractQreg() = qinsert d 0 k
+          o:AbstractQreg() = qinsert n 1 j
+           = qdealloc o
+        in (m,) }
+      fn=<QNode: wires=2, device='braket.local.qubit', interface='auto', diff_method='best'>
+    ] a
+  in (b,) }
+```
+
+To register ISWAP and PSWAP as supported for OpenQasm, they needed to be added to the utilities ` enum class Ligntning::SimulatorGate` and `std::array Ligntning::simulator_gate_info` in Utils.hpp. Because all the three NamedOperation() for the three devices share the same `Lightning::lookup_gates()` from common/Utils.hpp, and PSWAP and ISWAP are only supported on OpenQasm, the NamedOperation() for Lightning and LightningKokkos are modified so that they do not try to fetch any unsupported ISWAP and PSWAP.
+
+This issue fundamentally arises from the fact that the three devices share the same `lookup_gates()` from common/Utils.hpp in their respective `LightningSimulator/LightningKokkosSimulator/OpenQasmDevice::NamedOperation()`. 
+
+As a workaround, the ISWAP and PSWAP NamedOperation() for Lightning and LightningKokkos are right now hard-coded by their matrices explicitly. I suggest the removal of a shared utility array and switch to another kind of gate support in C++ simulator runtime to avoid similar issues in the future.
+  [(#458)](https://github.com/PennyLaneAI/catalyst/pull/458)
```

Comment:
Don't forget to add your name to the list of contributors :slightly_smiling_face: